src/utils/openUrl.ts

```diff
@@ -3,8 +3,14 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as vscode from 'vscode';
+// tslint:disable-next-line:no-require-imports
+import opn = require("opn");
 
 export async function openUrl(url: string): Promise<void> {
-    await vscode.env.openExternal(vscode.Uri.parse(url));
+    // Using this functionality is blocked by https://github.com/Microsoft/vscode/issues/25852
```

Comment:
> Instead of referencing a VS Code bug, I would reference a bug in our repo. Specifically saying which scenario doesn't work with 'vscode.open' so that we will know what to test if/when we want to move back to that instead of opn

😅

Reply:
I did!
> Specifically, opening the Live Metrics Stream for Linux Function Apps doesn't work in this extension.

Reply:
Ohhh, that's what you meant.  I thought you meant to reference an issue in our Github repo. 👍